app/scripts/modules/kubernetes/src/v2/pipelines/stages/findArtifactsFromResource/findArtifactsFromResourceStage.ts

```diff
@@ -0,0 +1,35 @@
+import { module } from 'angular';
+
+import {
+  PIPELINE_CONFIG_PROVIDER,
+  PipelineConfigProvider,
+  SETTINGS
+} from '@spinnaker/core';
+
+import { KubernetesV2FindArtifactsFromResourceConfigCtrl } from './findArtifactsFromResourceConfig.controller';
+import { KUBERNETES_MANIFEST_SELECTOR } from '../../../manifest/selector/selector.component';
+
+export const KUBERNETES_FIND_ARTIFACTS_FROM_RESOURCE_STAGE = 'spinnaker.kubernetes.v2.pipeline.stage.findArtifactsFromResource';
+
+module(KUBERNETES_FIND_ARTIFACTS_FROM_RESOURCE_STAGE, [
+  PIPELINE_CONFIG_PROVIDER,
+  KUBERNETES_MANIFEST_SELECTOR
+]).config((pipelineConfigProvider: PipelineConfigProvider) => {
+    // Todo: replace feature flag with proper versioned provider mechanism once available.
+    if (SETTINGS.feature.versionedProviders) {
```

Comment:
I would feature flag this on the "artifacts" feature flag instead.

Reply:
Done.